charts/cron-job/values.yaml

```diff
@@ -37,10 +37,12 @@ config:
 # -- List of config maps to mount to the deployment
 configMaps: []
 #  - name: "volume-1"
-#    key: "file.cfg"
-#    contentsB64: "" # The file contents which have already been base-64 encoded
 #    mountPath: "/etc/config" # Must be unique
 #    mountPropagation: None # If unset will default to 'None'
+#    files: []
```

Comment:
should this be uncommented by default?

Reply:
So by default we do not mount any files - the user should specify this - this is purely for show on how they can implement this value.

Reply:
got it!